docs/notebook_tutorial/ocr.ipynb

```diff
@@ -0,0 +1,535 @@
+      "text/plain": [
+       "<IPython.core.display.Image object>"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {
+      "image/jpeg": {
+       "width": 250
+      }
+     },
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import IPython\n",
+    "image_url = 'https://raw.githubusercontent.com/asyml/forte/assets/ocr_tutorial/ocr.jpg'\n",
+    "IPython.display.Image(image_url, width = 250)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's set up a forte pipeline for the OCR task."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
```

Comment:
The output list is a bit too long. Maybe we should only show a few characters.

Another interesting idea is, maybe we can add a simple NLTKTokenzier to get the tokens, and then:

```python
for token in pack.get(Token):
    charcters = list(token.get(Character))
    top_left = pack.get_links_by_parent(charcters[0])[0].corners[0]
    bottom_right = pack.get_links_by_parent(charcters[-1])[0].corners[-1]

    # This step can be done in a processor, but can also be here.
    box = pack.add_entry(Box(top_left, bottom_right)) 
    # Visuliaze the bounding box (image with a box around it).
    print(token.text)
    box.visualize()         
```

Reply:
Added as suggested.